idd/Energy+.idd.in

```diff
@@ -89227,21 +89227,67 @@ HybridModel:Zone,
        \type object-list
        \object-list ZoneNames
   A3 , \field Calculate Zone Internal Thermal Mass
-       \note Use measured temperature data to calculate zone temperature capacity multiplier
-       \note Zone Air Infiltration Rate and Zone Internal Thermal Mass calculations cannot be performed simultaneously
+       \note Use measured data (temperature, humidity ratio, or CO2 concentration) to calculate zone temperature capacity multiplier.
+       \note At leat one measured parameter should be provided to calculate the thermal mass.
```

Comment:
Should be noted like: If A3 - Calculate Zone Internal Thermal Mass is defined, at least one measured parameter, defined in A6, A7, or A8 should be provided.